model_cards/rohanrajpal/bert-base-multilingual-codemixed-cased-sentiment/README.md

```diff
@@ -0,0 +1,93 @@
+---
+language: "multilingual"
```

Comment:
```suggestion
language:
- hi
- en
```

If this model is intended to use mostly for code-mixed HI+EN content, I'd suggest using this metadata.

What do you think?

Reply:
Sure, doing it right away. Thanks :)